<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattleWindow</class>
 <widget class="QMainWindow" name="BattleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>迷失的坦克beta0.0.4</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="Bar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>50</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLabel" name="showhp">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>HP:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="Bar_2">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>100</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="Bar_3">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>150</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="Bar_4">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>200</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>MP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>110</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>WeaponLv:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>160</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Matrial:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>220</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Process:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="Bar_5">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>260</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>690</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>升级子弹数量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>740</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>升级子弹等级</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>790</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>维修坦克</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
